### PR TITLE
OpenAI/ChatGPT Question

### PR DESCRIPTION
**Pull request by shaunxp20**
This PR will:
- Add a new migration for the new OpenAI/ChatGPT question
- Add the new question to the submission flow (including the submission review pages)

The admin piece for this will come later. 👍

### CODE REVIEW

File: app/helpers/boolean_helper.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    case boolean
+    when true
+      "Yes"
+    when false
+      "No"
+    when nil
+      "-"
+    end
+  end
+end
```
Comment (shaunxp20) [inline app/helpers/boolean_helper.rb:12]: A new helper to help us convert true/false/nil to Yes/No/-, inspired by this [SO answer](https://stackoverflow.com/a/57301785).

Comment (viviancan) [reply app/helpers/boolean_helper.rb:12]: Nice helper 👍

File: config/locales/views/submissions/en.yml [hunk 2]
```diff
@@ -1,3 +1,4 @@
 en:
   submissions:
     solve_hunger_or_food_waste_question: Does your submission help solve hunger or food waste in some way?
+    uses_open_ai_question: "Did you use OpenAI/ChatGPT at any point when working on your project?"
```
Comment (viviancan) [inline config/locales/views/submissions/en.yml:4]: Great to see this in action again!

**Review by viviancan — APPROVED** [review]
Looks good to me! Thanks for tackling this, Shaun 💯

**Pull request merged**